Documentation/ExtensionArchitecture/Extbase/Introduction/Index.rst

```diff
@@ -0,0 +1,93 @@
+.. include:: /Includes.rst.txt
+.. index::
+   pair: Extension development; Extbase introduction
+
+.. _extbase-introduction:
+
+====================
+Extbase introduction
+====================
+
+.. todo:: Refer to a good example, e.g. tea, blog_example?
+
+.. todo:: Add tips how to get more information or help
+
+What is Extbase?
+================
+
+Extbase is provided via a TYPO3 system extension (typo3-cms/extbase). Extbase is
+a framework within the TYPO3 Core. The framework is based on the software pattern
+`MVC (model-view-controller) <https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller>`__
+and uses
+`ORM (object relational modeling) <https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping>`__.
+
+Extbase can be and is often used in combination with the Fluid templating engine,
+but Fluid **can** also be used without Extbase. All things that can be done with
+Extbase, can also be done with TYPO3 Core native functionality.
+Extbase is not a prerequisite for extension development. In most cases, not
+using Extbase means writing more code. In many cases the performance will be
```

Comment:
"In most cases, not
 using Extbase means writing more code" => Please do not use negations. "In most cases, using Extbase means writing less code.

Reply:
Good point, will fix